src/ontology/oba.Makefile

```diff
@@ -81,12 +81,15 @@ $(MIRRORDIR)/lipidmaps.owl: $(TEMPLATEDIR)/lipidmaps.tsv
 $(ONT)-full.owl: $(SRC) $(OTHER_SRC)
 	echo "INFO: Running FULL release, which is customised for OBA."
 	$(ROBOT) merge --input $< \
-		merge -i components/reasoner_axioms.owl \
-		materialize -T basic_properties.txt \
-		reason --reasoner ELK --equivalent-classes-allowed all --exclude-tautologies structural \
+		reason --reasoner ELK --equivalent-classes-allowed asserted-only --exclude-tautologies structural \
+		remove --term PR:000000001 \
+			   --term SO:0000252 \
+			   --term SO:0000234 \
```

Comment:
We remove a number of asserted equivalent classes forcefully. All three terms have equivalent classes that we are preserving.